src/forms/TopComponent/Utils/TitleWrapper.tsx

```diff
@@ -11,25 +11,25 @@ const BIG_SCREEN_BREAKPOINT = BREAKPOINTS.topComponent.bigScreenView + "px";
 
 export const FormTitle = styled.span`
 	font-family: ${pr => !pr.type && theme.museoFont};
-	font-weight: ${pr => pr.type && pr.type === "drawer" && "medium"};
+	font-weight: ${pr => pr.type && pr.type === "DRAWER" && "medium"};
 
-  color: ${theme.colors.almostBlack};
-  font-size: ${pr => pr.type && pr.type === "drawer" ? "20px" : "28px"};
-  margin: ${pr => pr.type && pr.type === "drawer" ? "auto" : ""};
+	color: ${theme.colors.almostBlack};
+	font-size: ${pr => pr.type && pr.type === "DRAWER" ? "20px" : "28px"};
+	margin: ${pr => pr.type && pr.type === "DRAWER" && "auto"};

```

Comment:
I believe here, if type !== "DRAWER" the margin will be something like margin: undefined; which won't "really" affect the css but shouldn't be there. 

I suggest if possible, changing to something like: ${pr => pr.type === "DRAWER" && magin: auto; font-weight: "medium";}
 (you could add the font-weight here as they deal with the same logic)